app/src/main/java/com/android/streetworkapp/ui/profile/Profile.kt

```diff
@@ -33,47 +41,130 @@ fun ProfileScreen(
     userViewModel: UserViewModel = viewModel(factory = UserViewModel.Factory),
     innerPaddingValues: PaddingValues = PaddingValues(0.dp)
 ) {
+
+  val context = LocalContext.current
+
+  // Fake data extraction from the MVVM :
+  val alice =
+      User("uid_Alice", "Alice", "alice@gmail.com", 42, listOf("Z8qgzNOoQOR09x6QQYahcyzVXfE2"))
+
+  val bob = User("uid_Bob", "Bob", "bob@gmail.com", 64, emptyList())
+
+  val currentUser =
+      User("uid_current", "Current User", "user@gmail.com", 42424, listOf(alice.uid, bob.uid))
+
+  var friendList = listOf(alice, bob)
+  // friendList = emptyList()
+
   Box(modifier = Modifier.testTag("ProfileScreen")) {
     Column(
         modifier = Modifier.fillMaxSize().padding(innerPaddingValues).testTag("ProfileColumn"),
         horizontalAlignment = Alignment.CenterHorizontally,
         verticalArrangement = Arrangement.spacedBy(10.dp, Alignment.CenterVertically)) {
-          // profile placeholder
 
-          Image(
-              painter = painterResource(id = R.drawable.profile),
-              contentDescription = "profile picture",
-              modifier = Modifier.size(200.dp))
+          Spacer(modifier = Modifier.height(4.dp).testTag("profileSpacer"))
```

Comment:
Question: I am not sure why this spacer in particular is used? Is the space above the picture profile not enough?

Reply:
Before the scaffold merge it was needed to space things more specifically than simply using padding. I removed it.

Reply:
I also removed the others spacers and instead put small padding to prevent things from breaking on other formats